spec/features/idv/doc_auth/document_capture_step_spec.rb

```diff
@@ -86,8 +132,6 @@
         'IdV: ' + "#{Analytics::DOC_AUTH} document_capture submitted".downcase,
         step: 'document_capture',
         flow_path: 'standard',
-        doc_auth_result: 'Passed',
-        billed: true,
```

Comment:
Yet another bug surfaced by this transition to opting-in JavaScript for these steps: We aren't and haven't been logging these details in real-world usage. From what I can tell the reports don't really seem to depend on these details, and I assume they've been missing for a while, but figured to raise just-in-case, cc @stevegsa @zachmargolis (via git blame'd #4018).

Reply:
We're not billing for individual calls to vendors anymore, you're right. But if it's possible to wire these back in, I think that would be good. Or at least file a JIRA so we can get rid of that stuff entirely

Reply:
Happy to make a ticket, but I'm not sure the full extent of what we're okay to remove. Does that include proofing costing details ([example](https://github.com/18F/identity-idp/blob/main/app/services/db/add_document_verification_and_selfie_costs.rb)) ?

Reply:
Oh wait, those I think may be used... I will check in Slack

Reply:
Actually, that code may be working as expected. Kinda hard to follow all the layers of log data 😩 There is a "billed" property on the actual vendor calls that is present, but for the purposes of the log tested here previously, was not being passed along into the "document_capture submitted" event.